modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -28,7 +28,20 @@ def perform(uuid)
         user = IAMUser.find(uuid) || User.find(uuid)
         raise MissingUserError, uuid unless user
 
+        mobile_user = Mobile::User.find_or_create_by(icn: user.icn)
+
+        if mobile_user.vet360_linked
+          Rails.logger.info('This User previously had a vet360 ID linked',
+                            { icn: mobile_user.icn, attempts: mobile_user.vet360_link_attempts })
+          mobile_user.vet360_linked = false
+        end
+
+        mobile_user.increment_vet360_link_attempts
+
         result = VAProfile::Person::Service.new(user).init_vet360_id
+
+        mobile_user.save
```

Comment:
If the external linking call fails due to networking issues, I'd speculate this save doesn't happen. Which is probably what we want because the attempt wasn't successful. 👍

Reply:
Yes it should fail to ever hit the save in the event of a failure